Symfony/src/AppBundle/Entity/User.php

```diff
@@ -32,6 +38,11 @@ class User implements UserInterface, \Serializable
      * @var string
      *
      * @ORM\Column(name="email", type="string", length=255, unique=true)
+     *
+     * @Assert\Length(
```

Comment:
I guess you mean the existing check when the user registers is performed in the frontend/form. But the entity doesn't only get its data from the frontend; the backend could send them in somehow in the future (for example from an import or external registration). So the base check is always necessary to make sure your data are consistent. An extra check where it is needed (like the form input) is better than getting a Doctrine error but if that fails, getting the Doctrine error will save the day.